tests/ui/specs/PopupTabLayout/qa-sampler/PopupTabLayout-QWTC-2008-specs.js

```diff
@@ -0,0 +1,49 @@
+const Page = require('../PopupTabLayoutPage');
+const {getFocusedText} = require('../../utils');
+
+describe('qa sampler', function () {
+	const {popupTabLayout} = Page.components;
+
+	beforeEach(async function () {
+		await Page.open('WithVariousItems');
+	});
+
+	describe('PopupTabLayout', function () {
+		it('should navigate with left and right key [QWTC-2008]', async function () {
```

Comment:
why did you add QWTC-2008 ? It  is  a test case for VirtualList

Reply:
Yes, I automated some existing test cases and these are their IDs